modules/android/pages/_partials/_define_module_attributes.adoc

```diff
@@ -6,6 +6,12 @@
 // SET full maintennce version number
 :version-maintenance: {version}.{maintenance-android}{tag}
 :version-maintenance-hyphenated: {major}-{minor}-{maintenance-android}{tag}
+
+// VECTOR SEARCH attributes
+:vs-version-maintenance: {vs-version}.{vs-maintenance-android}{tag}
```

Comment:
It's a bit odd that you use {vs-version} here, but {vs-major}-{vs-minor} directly below.

Reply:
It's the hyphens and to mirror the hyphenated version above. I'm not too sure *when* exactly this is used as it's probably an edge case but added it in the off chance I'll need it at some point.